check_repo.py

```diff
@@ -19,7 +19,12 @@ def check_repo():
         os.path.dirname(os.path.realpath(__file__)), os.pardir))
     print("Repo path " + repo_path)
 
-    toplevel_folders = sorted(next(os.walk(repo_path))[1])
+    parameters = sys.argv[1:]
+    if len(parameters) == 0:
+        toplevel_folders = sorted(next(os.walk(repo_path))[1])
+    else:
+        toplevel_folders = sorted(parameters)
```

Comment:
maybe build the intersection of actuall toplevel_folders and parameters to be more fail safe

Reply:
Talked with rechi and this is not necessary